include/CLI/StringTools.hpp

```diff
@@ -347,7 +347,7 @@ inline std::vector<std::string> split_up(std::string str, char delimiter = '\0')
 
     const std::string delims("\'\"`");
     auto find_ws = [delimiter](char ch) {
-        return (delimiter == '\0') ? (std::isspace<char>(ch, std::locale()) != 0) : (ch == delimiter);
+        return (delimiter == '\0') ? (static_cast<int>(std::isspace<char>(ch, std::locale())) != 0) : (ch == delimiter);
```

Comment:
```suggestion
        return (delimiter == '\0') ? std::isspace<char>(ch, std::locale()) : (ch == delimiter);
```
I guess std::isspace<TYPE> returns a bool directly,  no reason to convert it to int only to convert to a bool again.

Reply:
The regular one is int, but the locale one is bool. 👍